### ISSUE TITLE
Nukleus

### ISSUE THREAD

**Issue by anjamaa** [realfagstermer/realfagstermer#1034]
Katalogpostene handler om cellekjerner, slå sammen med `Cellekjerner`?

--
URI: http://data.ub.uio.no/realfagstermer/c001008
Bruk: http://ub-viz01.uio.no/okapi2/#/search?q=realfagstermer%3A%22Nukleus%22
Soksed: https://soksed.biblionaut.net/concepts/REAL001008

**Comment by Hypsibius** [comment]
Ja, med cellekjerner som foretrukken term. Skjønt, kanskje treng vi spesifisere til Cellenukleus i BF-termen?

**Comment by danmichaelo** [comment]
Ok, enda det, jeg. Greit å følge ordboka, og det dukker jo opp som forslag hvis en søker etter "nukleus" også.
